aztk/client/base/helpers/delete_user_on_node.py

```diff
@@ -0,0 +1,9 @@
+def delete_user(self, pool_id: str, node_id: str, username: str) -> str:
+    """
```

Comment:
doesn't looks like to be too much of those left. Could change them all now to the new format

Reply:
This docstring is not published. I can update it, but it will only be discoverable in the source.